libs/hiraya/src/components/Button/theme.ts

```diff
@@ -0,0 +1,43 @@
+const variants = {
+  primary: {
+    background: '#305EFF',
+    text: 'white',
+    border: 'none',
+    hoverBackground: '#2950DA',
+  },
+  secondary: {
+    background: '#FFFFF',
+    text: '#305EFF',
+    border: '1px solid #305EFF',
+    hoverBackground: '#9FA8DA',
+  },
+  tertiary: {
+    background: '#BDBDBD',
+    text: '#333333',
+    border: 'none',
+    hoverBackground: '#9E9E9E',
+  },
+};
+
+const buttonSizesStyle = {
+  small: {
```

Comment:
Instead of trying to set a specific height and width, can we use a bigger fontSize instead? and then let the padding bigger that way we dont set a specific height and width that will be hard for the consumer to designo or use.

Reply:
It doesn't matter if the design is not 1:1, especially for height and width, I think for now what matters more is the extensibility of the button, what I mean is that it would be easier to extend the button and add designs to it via for example I want the button the fill my container, like that kind of stuff but with you setting the height and width on the ui level, I won't be able to do that when I use this button.